shared-libs/purging-utils/src/index.js

```diff
@@ -0,0 +1,33 @@
+const crypto = require('crypto');
+
+const sortedUniqueRoles = roles => ([...new Set(roles)].sort());
+const getRoleHash = roles => crypto
+  .createHash('md5')
+  .update(JSON.stringify(sortedUniqueRoles(roles)), 'utf8')
+  .digest('hex');
+
+const getPurgeDbName = (mainDbName, rolesHash) => `${mainDbName}-purged-role-${rolesHash}`;
+
+// todo consider moving this somewhere else, it's a duplicate of a function that atm exists in the users controller in api
```

Comment:
A new user shared-lib might work, but feel free to raise a separate issue for this.

Reply:
Will do after that other PR is merged.